addons/base_automation/data/digest_data.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <data>
+        <record id="digest_tip_base_automation_0" model="digest.tip">
+            <field name="name">Tip: Automate everything with Automation Rules</field>
+            <field name="sequence">3700</field>
+            <field name="group_id" ref="base.group_system"/>
+            <field name="tip_description" type="html">
+<div>
+    <p class="tip_title">Tip: Automate everything with Automation Rules</p>
+    <p class="tip_content">Send an email when an object changes state, archive records after a month of inactivity or remind yourself to follow-up on tasks when a specific tag is added.<br/>With Automation Rules, you can automate any workflow.</p>
+    <img src="/digest/static/src/img-v18/18-automation-rules.gif" width="540" class="illustration_border"/>
```

Comment:
Maybe I'm stupid, but the image you refer to is specific to this module.  Why put it in module `digest`?

Reply:
All the gifs and images are temporarily set in a folder in digest as I'm waiting for them to be hosted on odoo cdn like all the other digest tips :slightly_smiling_face: 
Example: https://download.odoocdn.com/digests/crm/static/src/img/milk-pipeline-progress.gif
Once they're hosted all the `src`'s will be updated :smile: